src/Build.UnitTests/Instance/ProjectInstance_Internal_Tests.cs

```diff
@@ -553,6 +558,116 @@ public void Build()
             mockLogger.AssertLogContains(new string[] { "Building...", "Completed!" });
         }
 
+        [Theory]
+        [InlineData(
+            @"      <Project>
+                    </Project>
+                ")]
+        [InlineData(
+            @"      <Project InitialTargets=`t1` DefaultTargets=`t2` ToolsVersion=`{0}`>
```

Comment:
Smaller case that's easier to debug in case things blow up :)
Should I just delete it and leave the more complex project?